easybuild/easyconfigs/c/Cython/Cython-0.26-GCCcore-6.4.0-Python-2.7.13-bare.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '0.26'
+pyver = '2.7.13'
+pyshortver = '2.7'
+pysubver = '-bare'
+versionsuffix = '-Python-%s%s' % (pyver, pysubver)
```

Comment:
including `-Python-<version>` in the `versionsuffix` is meant to indicate that Python is pulled in as a runtime dependency

That's not the case here (it's only a build dep), but it should be since Cython can't be used without having (the right) Python around?

@JackPerdue Can you clarify your intentions here?

I suspect the `%(pyver)s` and `%(pyshortver)s` templates didn't work when Python is included as a build dep?